Assets/Scripts/GameManager.cs

```diff
@@ -81,13 +118,24 @@ public class GameManager : MonoBehaviour {
         gameOver = (players.Count == 1);
     }
 
-    void SetHealthTexts() {
-        if (player != null) {
-            playerHealthText.text = "Health: " + player.GetComponent<TankStats>().GetStat(StatType.MaxHealth);
-        }
-        if (computerPlayer != null) {
-            computerPlayerHealthText.text =
-                "Health: " + computerPlayer.GetComponent<TankStats>().GetStat(StatType.MaxHealth);
+    void SetStatsUIs() {
+        foreach (KeyValuePair<GameObject, Text> playerStatsUI in _playerStatsUis) {
+            if (playerStatsUI.Key != null) {
```

Comment:
A quick Google search makes me think that a dictionary cannot have a `null` key.